### PR TITLE
Add emmetapp.app v1.0.0

### PR DESCRIPTION
**Pull request by pusewicz**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/emmetapp.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  version "1.0.0"
+  sha256 :no_check
+
+  url "https://s3.amazonaws.com/emmet.app/emmetapp.dmg",
+      verified: "s3.amazonaws.com/emmet.app/"
+  name "emmetapp"
+  desc "Window manager you dream about"
+  homepage "https://emmetapp.com/"
+
+  auto_updates true
```
Comment (neutric) [inline Casks/emmetapp.rb:11]:
```
livecheck do
  url "https://s3.amazonaws.com/emmet.app/appcast.xml"
  strategy :sparkle
end
```

File: Casks/emmetapp.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+cask "emmetapp" do
+  version "1.0.0"
+  sha256 :no_check
+
+  url "https://s3.amazonaws.com/emmet.app/emmetapp.dmg",
+      verified: "s3.amazonaws.com/emmet.app/"
+  name "emmetapp"
+  desc "Window manager you dream about"
```
Comment (neutric) [inline Casks/emmetapp.rb:8]: [Do not use empty marketing jargon.
](https://docs.brew.sh/Cask-Cookbook#dos-and-donts)
`desc "Tiling and stacking window manager, window resizing tool"`

File: Casks/emmetapp.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  homepage "https://emmetapp.com/"
+
+  auto_updates true
+  depends_on macos: ">= :catalina"
```
Comment (neutric) [inline Casks/emmetapp.rb:12]: > Which macOS versions does emmetapp support?
> Currently we're supporting macOS Catalina and Big Sur.
> 

```
depends_on macos: [
  :catalina,
  :big_sur,
]
```

**Comment by miccal** [comment]
Thank you @pusewicz.

File: Casks/emmetapp.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    :catalina,
+    :big_sur,
+  ]
+
+  app "emmetapp.app"
+
+  zap trash: [
+    "~/Library/Application Scripts/com.speedyhill.emmetapp-LaunchAtLoginHelper",
+    "~/Library/Containers/com.speedyhill.emmetapp-LaunchAtLoginHelper",
+    "~/Library/Group Containers/2BUA8C4S2C.com.agilebits",
```
Comment (pusewicz) [inline Casks/emmetapp.rb:27]: Actually, just saw that this line should be removed! I'll send a follow-up PR.

**Pull request merged**